coderd/database/migrations/000295_organization_soft_delete.up.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE organizations ADD COLUMN deleted boolean DEFAULT FALSE NOT NULL;
+
+DROP INDEX IF EXISTS idx_organization_name;
+DROP INDEX IF EXISTS idx_organization_name_lower;
+
+CREATE UNIQUE INDEX IF NOT EXISTS idx_organization_name ON organizations USING btree (name)
+	where deleted = false;
+CREATE UNIQUE INDEX IF NOT EXISTS idx_organization_name_lower ON organizations USING btree (lower(name))
+	where deleted = false;
```

Comment:
We only need 1 index on the name imo
```suggestion
CREATE UNIQUE INDEX IF NOT EXISTS idx_organization_name_lower ON organizations USING btree (lower(name))
	where deleted = false;
```

Reply:
Just to double check, both of these indexes were added in 000001_base.up.sql. Did you mean removing 1 of the indexes completely?

Reply:
Yes, we only need to include the `lower()` one.

Reply:
Unless we specifically do case sensitive searches.